docs/pages/includes/plugins/identity-export.mdx

```diff
@@ -1,31 +1,93 @@
-Teleport's plugins use the `access-plugin` user to manage access requests. In order to act as
-the user, the plugin must present the Teleport Auth Service with valid identity files.
-We export the identity files for the user using [`tctl auth sign`](../../setup/reference/cli.mdx#tctl-auth-sign).
+Like all Teleport users, `access-plugin` needs signed credentials in
+order to connect to your Teleport cluster. You will use the `tctl auth sign`
+command to request these credentials for your plugin. 
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+The format of the credentials depends on whether you have set up your network to
+give the plugin direct access to the Teleport Auth Service, or if all Teleport
+clients and services connect to the Teleport Proxy Service instead.
 
 <Tabs>
-  <TabItem label="Self-Hosted" scope={["oss","enterprise"]}>
+<TabItem label="Connect to the Proxy Service">
+
+
+The following `tctl auth sign` command impersonates the `access-plugin` user,
+generates signed credentials, and writes an identity file to the local
+directory:
+
+```code
+$ tctl auth sign --user=access-plugin --out=auth.pem
```

Comment:
The auth tab includes `--ttl=2190h`, should we include here as well? Or drop from both, we also "recommend" `--ttl=8760h` below

Reply:
Dropping both makes sense to me since we don't explain the use of the flag until that recommendation below